pkg/kubelet/kubelet.go

```diff
@@ -1799,16 +1799,26 @@ func (kl *Kubelet) syncTerminatingPod(ctx context.Context, pod *v1.Pod, podStatu
 		return err
 	}
 	var runningContainers []string
-	var containers []string
+	type container struct {
+		Name       string
+		State      string
+		ExitCode   int
+		FinishedAt string
+	}
+	var containers []container
+	klogV := klog.V(4)
```

Comment:
a simple comment like, _"verbosity level can be controlled by changing this line"_ might be helpful

Reply:
Done.